python/res/enkf/export/summary_collector.py

```diff
@@ -9,9 +9,9 @@
 class SummaryCollector(object):
 
     @staticmethod
-    def createActiveList(ert, fs):
+    def createActiveList(ensamble_size, fs):
```

Comment:
Unless you have a very good reason for doing so, we should not change the current API...

Reply:
There is a reason, if it's good enough is another matter. It's symptomatic for a lot of these functions to take in EnkFMain and use a long chain of commands on it, making it very hard to create new functionality without pulling EnkFMain down to the lowest level, when in this case what was actually needed was an int. Changing that as the functionality was used seemed a rational way to start stabbing at the problem, but might not be worth a breaking change.

Reply:
I see your reasoning, but let us save this for later... We should first make an evaluation of what we want the API to look like before we start making micro improvements and at the same time breaking our API.

Reply:
:+1: